Sources/RichTextKit/Component/RichTextViewComponent+Link.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  RichTextViewComponent+Link.swift
+//
+//
+//  Created by Dominik Bucher on 04.12.2023.
+//
+
+import Foundation
+
+/// This wont do anything since `linkAttributes` are not set to nil currently.
+/// Waiting for https://github.com/danielsaidi/RichTextKit/pull/142 to be merged to implement link configuration.
+fileprivate let linkColor = ColorRepresentable.green
+
+public extension RichTextViewComponent {
+    /// Unsets the current link at the given range.
+    /// This method takes the range, checks if there are any attributes for link and removes them.
+    /// After that, sets default typing attributes so we can continue writing as is.
+    /// Note if nothing is selected, we just set typing attributes so we can continue writing in normal pace
+    func unsetLinkFromCurrentRichTextStyle() {
```

Comment:
We no longer use the `current` term for the component, so I think renaming this to `removeLink` would be enough.